### PR TITLE
AST-2360 - fix: Getting console error in customizer when trying to update footer columns.

### PR DESCRIPTION
**Pull request by yssambare12**
**Steps To Reproduce:** 

1. Navigate to customizer > footer builder
2. Jump on any footer row primary/above/below. Try changing columns optio

**Actual Result:**

- Error - Uncaught RangeError: Maximum call stack size exceeded.

**Expected Result:** 

- There should not be any errors.

**Attachment:** 

- Screencast - https://share.bsf.io/bLu59XpG

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/extend-custom-controls/src/row-layout/row-layout-component.js [hunk 1]
```diff
@@ -139,7 +139,6 @@ class RowLayoutComponent extends Component {
 			value.tablet  = defaults[columns];
 			value.mobile  = 'full';
 			this.setState( { value : value } );
-			this.updateValues();
```
Comment (yssambare12) [inline inc/customizer/extend-custom-controls/src/row-layout/row-layout-component.js:142]: After event dispatch, still, again here value is updating; that's why this goes in an infinite loop and shows this call stack error.

**Comment by neerajmasai** [comment]
Can you please check the code analysis failure.

**Comment by neerajmasai** [comment]
Also please handle the conflicts with base branch

**Comment by yssambare12** [comment]
> Can you please check the code analysis failure.
@neerajmasai Resolved

**Comment by yssambare12** [comment]
> Also please handle the conflicts with base branch

@neerajmasai Updated as per base branch.

**Review by nathanrodrigues2111 — APPROVED** [review]

**Comment by yssambare12** [comment]
Closing This PR due to some unsolved critical conflicts 
& creating new PR for this same - https://github.com/brainstormforce/astra/pull/4789

**Pull request closed without merging**